srcpkgs/octave/template

```diff
@@ -1,19 +1,33 @@
 # Template file for 'octave'
 pkgname=octave
-version=5.2.0
-revision=2
+version=6.2.0
+revision=1
 build_style=gnu-configure
-configure_args=" --with-blas=openblas --with-lapack=openblas"
-hostmakedepends="perl gcc-fortran pkg-config gnuplot"
+hostmakedepends="perl gcc-fortran pkg-config gnuplot texinfo texlive-dvi tar"
 makedepends="pcre-devel openblas-devel readline-devel libSM-devel libltdl-devel
  lcms2-devel glpk-devel"
-depends="$(vopt_if gui qt5-plugin-sqlite) texinfo"
+depends="$(vopt_if gui qt5-plugin-sqlite) texinfo texlive-dvi tar"
```

Comment:
what for is the runtime dependency on tar? I'm inclined to not add it to `depends` as tar is part of base-system already (no other package explicitly depends on tar, though many might need it)

Reply:
Assuming it's necessary only for building packages, I don't think it's necessary. You still need `gcc-fortran` and other dependencies to build them.

Reply:
Sorry for dropping the ball on this. I'm currently too busy to work further on this. Anyone else feel free to pick up where I left off.